package.json

```diff
@@ -15,6 +15,7 @@
     "build": "./build.sh",
     "lint": "eslint --ext .js,.jsx .",
     "server": "node server.js",
+    "groupsOnly": "node server.js --app ./src/groupsOnlyApp.js",
```

Comment:
Thanks for the helpful perspective. This new entrypoint should be framed properly, and naming is a small, tangible way of framing.

Binning it as a component of nextstrain.org makes sense on the server side, but is misleading on the Gatsby side. More concretely, I think your [suggestion](https://github.com/nextstrain/nextstrain.org/pull/735#discussion_r1391529108) `s/groupsOnlyService/groupsServer/` is misleading because Groups is being served regardless of its `true`/`false`. Maybe something like `minimalGroupsServer`?

Reply:
Forgot to mention that I [applied](https://github.com/nextstrain/nextstrain.org/compare/49c70f42181e6c35158a4fda129d983f965fe52f..bed30dc0ff21f27a89fea035075228289037b436) the suggested server change here, but not the static-site change.

Reply:
> More concretely, I think your [suggestion](https://github.com/nextstrain/nextstrain.org/pull/735#discussion_r1391529108) `s/groupsOnlyService/groupsServer/` is misleading because Groups is being served regardless of its `true`/`false` value. Maybe something like `minimalGroupsServer`?

That's fair. My was thinking that `groupsServer: true/false` was less "Groups are/aren't being served" and more "this is/isn't a Groups Server instance". i.e. it's not a small, scoped feature flag but a whole different build/behaviour mode (e.g. similar to our `PRODUCTION` mode). In that light, maybe more natural to think about the inverse: "this is/isn't actual nextstrain.org", e.g. `nextstrainDotOrg: true/false`? Not sure! I could also see `groupsApp: true/false` being clearer due to tie in with `src/groupsApp.js`.

In any case, this is a minor thing. Go with what makes sense to you.

Reply:
I like `groupsApp`. Switched to it in [force-push](https://github.com/nextstrain/nextstrain.org/compare/e49ad0059aced63d7ef5d262fdb24bda11f56aef..e09081a870ce6a01415181ed62a9f51391d28f04).